src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,10 @@
-export const PostList = () => (
-  <>Put the list here</>
+import './PostList.scss';
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {
+      posts.map(post => <PostInfo post={post} key={post.id} />)
+    }
```

Comment:
```suggestion
    {posts.map(post => (
      <PostInfo post={post} key={post.id} />
    ))}
```
Formatting can be improved here